source/includes/_cluster.md

```diff
@@ -160,7 +164,7 @@ Property | Type | Description
 -------- | ---- | -----------
 host_ip | string | The externally accessible IP of this node.
 node_name | string | A unique name for this node to be used internally within the cluster.
-encryption_key | string | A base64-encoded 16-byte encryption key, identical for all the nodes in the cluster.
+credentials | string | When joining a node, provide the credentials received from the cluster master.
```

Comment:
received from the cluster ~master~ **active node**.

(We don't use the term "master")